ra2ce/graph/origins_destinations.py

```diff
@@ -144,19 +144,25 @@ def closest_node(node: np.ndarray, nodes: np.ndarray) -> np.ndarray:
     return nodes[np.argmin(dist_2)]
 
 
-def get_od(o_id, d_id):
+def get_od(o_id: str, d_id: str) -> str:
```

Comment:
```suggestion
def get_od(o_id: str, d_id: str) -> str | np.nan:
```

Reply:
Unfortunately this is not accepted by Python, not sure why, this however does get accepted (`| type("np.nan")`). I will explore this in the future after the corresponding SonarCloud analysis.